diff --git a/index.d.ts b/index.d.ts
index a018fbc7abf0fd51402cd70bd7bdb88a3fa7c866..d8d72bab7762b0c83e7c0440d00f02dde89a22e7 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -72,6 +72,13 @@ export type AnimationItem = {
     segmentPos: number;
     isSubframeEnabled: boolean;
     segments: AnimationSegment | AnimationSegment[];
+    markers: {
+        time: number;
+        duration: number;
+        payload: {
+            name: string;
+        };
+    }[];
     play(name?: string): void;
     stop(name?: string): void;
     togglePause(name?: string): void;
