diff --git a/dist/app/components/nuxt-link.js b/dist/app/components/nuxt-link.js
index ebfa404c80eac774a1102386e1aeb4efa929b500..b3bc3a114c536600ca3884018dd818b3d81d5c8a 100644
--- a/dist/app/components/nuxt-link.js
+++ b/dist/app/components/nuxt-link.js
@@ -251,9 +251,7 @@ export function defineNuxtLink(options) {
       }
       if (import.meta.dev && import.meta.server && !props.custom) {
         const isNuxtLinkChild = inject(NuxtLinkDevKeySymbol, false);
-        if (isNuxtLinkChild) {
-          console.log("[nuxt] [NuxtLink] You can't nest one <a> inside another <a>. This will cause a hydration error on client-side. You can pass the `custom` prop to take full control of the markup.");
-        } else {
+        if (!isNuxtLinkChild) {
           provide(NuxtLinkDevKeySymbol, true);
         }
       }
