diff --git a/lib/container.js b/lib/container.js
index 462e3f0d3944ed56d37403d95b229c2942f9e731..f43b141639c0c929d4fb236d3cc794689c867821 100644
--- a/lib/container.js
+++ b/lib/container.js
@@ -298,7 +298,6 @@ class Container extends Node {
       try {
         result = callback(child, i)
       } catch (e) {
-        throw child.addToError(e)
       }
       if (result !== false && child.walk) {
         result = child.walk(callback)
diff --git a/lib/node.js b/lib/node.js
index d79dd56e54d544bb906ab0b5b67ff252155b3068..80502a8b7cc40961129f6471b692bbf8f5d43778 100644
--- a/lib/node.js
+++ b/lib/node.js
@@ -55,7 +55,6 @@ class Node {
   }
 
   addToError(error) {
-    error.postcssNode = this
     if (error.stack && this.source && /\n\s{4}at /.test(error.stack)) {
       let s = this.source
       error.stack = error.stack.replace(
