diff --git a/node_modules/postcss/lib/container.js b/node_modules/postcss/lib/container.js
index 462e3f0..f43b141 100644
--- a/node_modules/postcss/lib/container.js
+++ b/node_modules/postcss/lib/container.js
@@ -298,7 +298,6 @@ class Container extends Node {
       try {
         result = callback(child, i)
       } catch (e) {
-        throw child.addToError(e)
       }
       if (result !== false && child.walk) {
         result = child.walk(callback)
diff --git a/node_modules/postcss/lib/node.js b/node_modules/postcss/lib/node.js
index d79dd56..80502a8 100644
--- a/node_modules/postcss/lib/node.js
+++ b/node_modules/postcss/lib/node.js
@@ -55,7 +55,6 @@ class Node {
   }
 
   addToError(error) {
-    error.postcssNode = this
     if (error.stack && this.source && /\n\s{4}at /.test(error.stack)) {
       let s = this.source
       error.stack = error.stack.replace(
