diff --git a/dist/vue-router.d.ts b/dist/vue-router.d.ts
index 8d44447ddad9cb43386e376a270bc4c1cbbc4a86..32cac33542c83e2ae66233a1e40b4d660aad752a 100644
--- a/dist/vue-router.d.ts
+++ b/dist/vue-router.d.ts
@@ -180,7 +180,7 @@ export declare interface LocationAsRelativeRaw {
  *
  * @public
  */
-export declare type LocationQuery = Record<string, LocationQueryValue | LocationQueryValue[]>;
+export declare type LocationQuery = Record<string, LocationQueryValue>;
 
 /**
  * Loose {@link LocationQuery} object that can be passed to functions like
@@ -638,7 +638,7 @@ export declare type RouteLocationRaw = string | RouteLocationPathRaw | RouteLoca
 export declare interface RouteMeta extends Record<string | number | symbol, unknown> {
 }
 
-export declare type RouteParams = Record<string, RouteParamValue | RouteParamValue[]>;
+export declare type RouteParams = { slug: RouteParamValue[] } & Record<string, RouteParamValue>;
 
 export declare type RouteParamsRaw = Record<string, RouteParamValueRaw | Exclude<RouteParamValueRaw, null | undefined>[]>;
 
